devops/charts/bg-monitoring/templates/grafana/deployment.yaml

```diff
@@ -0,0 +1,64 @@
+{{ if .Values.grafana.enabled }}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:
+{{- with .Values.grafana.deploymentAnnotations }}
```

Comment:
Shouldn't "with" be above "annotations"?

Reply:
no. It gets all annotations from env.yaml or values yaml if they exist